projeto-labeX/labe-x/src/bases/Bases.js

```diff
@@ -0,0 +1,4 @@
+
+
+export const url = "https://us-central1-labenu-apis.cloudfunctions.net/labeX/claudia-jackson"
+
```

Comment:
Armazenar a url numa variável facilita bastante pra quem for ler o código e para o programador 👏🏻 🥇